### DISCUSSION TITLE
Drop Requests App: multiple domains allowed inside the same JSON object?

### DISCUSSION THREAD

**Discussion by shalafi99** [TechnitiumSoftware/DnsServer#988] [Q&A] [answered]
Hello,

I would like to include a bunch of domain names inside the drop requests app.
My home network has a corporate-owned device with WPAD not configured in an optimal way and because of that, the device keeps sending public DNS requests (which my DNS Server running Technitium receives) resulting in a large amount of "NX Domain" responses.

I do not have admin rights in the device to try to fix the improper behavior myself and instead got interested in having the DNS Server simply drop the requests instead of returning NX Domain.

I tried including those domain names inside the same object containing the "example.com" domain (which is included by default in the Drop Requests App) like so:

"blockedQuestions": [
{
"name": "example.com",
"name": "wpad.corpdomain1.com",
"name": "wpad.corpdomain2.com",
"blockZone": true
},
]

But that did not work (more so, queries for the example.com domain stopped being dropped by DNS Server).

The only proper way to go about this setup I need would be to create one object of type {"blockZone": true} for each domain name?
There are 20+ of them (unfortunately, the domain suffix search list on the device is that long!), I tried it like described above hoping for a shorter length config file.

Thank you

**Comment by ShreyasZare** [comment, marked as answer]
Thanks for asking. In JSON format syntax, you cannot have same name for more than one property so the config you have here has syntax issue. To add more than one entry, you need to construct one JSON object per domain name.

Alternatively, you can just click on the NXDOMAIN legend on the dashboard to hide it if that is just the concern.

**Reply by shalafi99** [reply]
Thanks for confirming it.
I will add a new JSON object entry for each domain name I want the DNS Server to drop requests for then.

And thanks for the suggestion, but the place where I'd like to not see that large amount of NXDOMAIN responses anymore is inside the Query Logs App instead of the dashboard.

I understand those entries are essentially causing the log database to get larger and at the same time polluting queries I run (with data I do not want to see and which I cannot do anything about).

Regards!
